src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrect2.java

```diff
@@ -64,29 +69,35 @@
     InnerInputJavadocParagraphIncorrect anon = new InnerInputJavadocParagraphIncorrect() {
 
             /**
-         * <p>Some Javadoc. // violation
+         * <p>Some Javadoc. // violation 'Javadoc content should start from the next line after
+         * /\*\*.'
          *
-         * Some Javadoc. // violation above
+         * Some Javadoc. // violation above 'Javadoc content should start from the next line after
+         * /\*\*.'
          *
          * @since 8.0
          */
         public static final byte NUL = 0;
 
         /**
-         * /WARN/  Some Javadoc.<p> // violation
+         * /WARN/  Some Javadoc.<p> // violation 'Javadoc content should start from the next line
+         * after /\*\*.'
          *
          *  <p>  Some Javadoc.
          *
          * @see <a href="http://www.gwtproject.org/doc/latest/DevGuideOrganizingProjects.html#DevGuideModules">
-         *     Documentation about <p> GWT emulated source</a> // violation
+         *     Documentation about <p> GWT emulated source</a> // violation 'Javadoc content
+         *     should start from the next line after /\*\*.'
          */
         boolean emulated() {return false;}
 
         /**
-         * Double newline. // violation below
+         * Double newline. // violation below 'Javadoc content should start from the next line
+         * after /\*\*.'
          *
          *
-         * Some Javadoc. //DOUBLE WARN AT TWO PREVIOUS LINES // violation above
+         * Some Javadoc. //DOUBLE WARN AT TWO PREVIOUS LINES // violation above 'Javadoc content
+         * should start from the next line after /\*\*.'
```

Comment:
`// violation` should be the first comment for the parser to recognize it as a violation, having it as a second comment won't work.